cli/azd/pkg/infra/provisioning/provider.go

```diff
@@ -135,13 +136,24 @@ func NewProvider(
 	projectPath string,
 	infraOptions Options,
 	prompters Prompters,
+	alphaFeatureManager *alphafeatures.AlphaFeatureManager,
```

Comment:
This seems so un-`go`-like.

What if we had: `features *alpha.Features`?

Reply:
What's what is not good?  the `manager` label?  
I am following the strategy we have in most commands where managers are injected and named as `manager`.

Reply:
I'm fine with the `Manager` suffix (I agree it is somewhat un-go like but we use it a lot).  I do think a name like `alpha.FeatureManager` would be better than what we have (and in line with many of our other names)

Reply:
updated